modules/realvuAnalyticsAdapter.js

```diff
@@ -0,0 +1,84 @@
+// RealVu Analytics Adapter
+import adapter from 'src/AnalyticsAdapter';
+import adaptermanager from 'src/adaptermanager';
+import CONSTANTS from 'src/constants.json';
+// const utils = require('src/utils');
+
+var realvuAnalyticsAdapter = adapter({
+  global: 'realvuAnalytics',
+  handler: 'on',
+  analyticsType: 'library',
+  url: '//ac.realvu.net/realvu_boost.js'
+});
+
+var options = { };
+
+var top1 = window;
+try {
+  var wnd = window;
+  while ((top != top1) && (typeof (wnd.document) != 'undefined')) {
+    top1 = wnd;
+    wnd = wnd.parent;
+  }
+} catch (e) { };
+top1.boost_fifo = top1.boost_fifo || [];
+top1.boost_units = top1.boost_units || [];
+
+realvuAnalyticsAdapter.originEnableAnalytics = realvuAnalyticsAdapter.enableAnalytics;
+
+realvuAnalyticsAdapter.enableAnalytics = function (config) {
+  var msg = document.getElementById('msg_an');
+  if (msg) {
+    msg.innerHTML += 'config:<br>' + JSON.stringify(config) + '<br>';
+  }
+  options = config.options;
+  realvuAnalyticsAdapter.originEnableAnalytics(config);
+};
+
+realvuAnalyticsAdapter.track = function ({eventType, args}) {
+  var msg = document.getElementById('msg_an');
+  if (msg) {
+    msg.innerHTML += 'track: ' + eventType + '; ';
+  }
+  if (eventType === CONSTANTS.EVENTS.AUCTION_INIT) {
+    if (options.partner_id) {
+      var hb = $$PREBID_GLOBAL$$;
```

Comment:
Using `$$PREBID_GLOBAL$$` in adapters is not allowed. your events should be tracking the data emitted by each event.